python/mleap/sklearn/feature_extraction/text.py

```diff
@@ -15,31 +15,49 @@
 # limitations under the License.
 #
 
-from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import CountVectorizer, TfidfVectorizer
+from mleap.sklearn.pipeline import SimpleSerializer as PipelineSerializer
 from mleap.bundle.serialize import MLeapSerializer
 import uuid
 
 
-def serialize_to_bundle(self, path, model_name):
-    serializer = SimpleSerializer()
+def count_vectorizer_serialize_to_bundle(self, path, model_name):
+    serializer = CountVectorizerSerializer()
     return serializer.serialize_to_bundle(self, path, model_name)
 
 
-def mleap_init(self, input_features, prediction_column):
+def count_vectorizer_mleap_init(self, input_features, prediction_column):
     self.input_features = input_features
     self.prediction_column = prediction_column
     self.name = "{}_{}".format(self.op, uuid.uuid4())
 
 
 setattr(CountVectorizer, 'op', 'tokenizer')
-setattr(CountVectorizer, 'mlinit', mleap_init)
-setattr(CountVectorizer, 'serialize_to_bundle', serialize_to_bundle)
+setattr(CountVectorizer, 'mlinit', count_vectorizer_mleap_init)
+setattr(CountVectorizer, 'serialize_to_bundle', count_vectorizer_serialize_to_bundle)
 setattr(CountVectorizer, 'serializable', True)
 
 
-class SimpleSerializer(MLeapSerializer):
+def tfidf_vectorizer_serialize_to_bundle(self, path, model_name):
```

Comment:
Perhaps you can take a look at https://github.com/combust/mleap/blob/master/python/mleap/sklearn/preprocessing/data.py#L74 and https://github.com/combust/mleap/blob/master/python/mleap/sklearn/preprocessing/data.py#L155 and see what you think about how we've made mleap_init and serialize_to_bundle to support multiple serializers and maybe follow the same pattern?

Reply:
Will do!